### PR TITLE
Reduce memory leak for commands: oo, doo

### PR DESCRIPTION
**Pull request by gtors**
#4886

### CODE REVIEW

**Comment by radare** [comment]
let’s see what travis have to say about this :)

thanks!

File: libr/bin/bin.c [hunk 1]
```diff
@@ -1036,20 +1026,14 @@ static RBinObject *r_bin_object_new(RBinFile *binfile, RBinPlugin *plugin, ut64
...
 	r_buf_free (binfile->buf);
 	binfile->buf = r_buf_new ();
 	r_buf_set_bytes (binfile->buf, bytes, sz);
-#endif
 	return binfile->buf != NULL;
 }
 
 static RBinFile *r_bin_file_new(RBin *bin, const char *file, const ut8 *bytes, ut64 sz, ut64 file_sz, int rawstr, int fd, const char *xtrname, Sdb *sdb) {
 	RBinFile *binfile = R_NEW0 (RBinFile);
-	if (!binfile) return NULL;
```
Comment (gtors) [inline libr/bin/bin.c]: oops

Comment (gtors) [inline libr/bin/bin.c]: oops

**Comment by radare** [comment]
i thikn the code you are fixing was related to rabin2 -C to create tiny binaries. there are tests for this so i assume they are passing as long as travis seems green

**Comment by radare** [comment]
wait, i think this pr is not in travis because of the github downtime :(

**Comment by radare** [comment]
thanks! lgtm in localhost merged

**Pull request closed without merging**